components/selection/selection-action.js

```diff
@@ -47,55 +43,47 @@ class Action extends LocalizeCoreElement(SelectionSubscriberMixin(ButtonMixin(Rt
 		`;
 	}
 
-	get selectionInfo() {
-		return super.selectionInfo;
-	}
-
-	set selectionInfo(value) {
-		super.selectionInfo = value;
-		this.nonInteractive = (this.requiresSelection && this.selectionInfo.state === SelectionInfo.states.none);
+	constructor() {
+		super();
+		this._selectionInfo = new SelectionInfoController(this);
```

Comment:
This is how controllers work -- you `new` them in the constructor and pass them a reference to the host element and any additional options.